assembl/static2/js/app/components/cookies/cookiesSelector.jsx

```diff
@@ -0,0 +1,63 @@
+// @flow
+// eslint-disable
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Button } from 'react-bootstrap';
+import classnames from 'classnames';
+import CookieToggle from './cookieToggle';
+import type { CookiesObject } from './cookiesSelectorContainer';
+
+type Props = {
+  activeKey: ?string,
+  show: boolean,
+  cookies: ?CookiesObject,
+  handleCategorySelection: Function,
+  handleSave: Function,
+  handleToggle: Function
+}
+
+const CookiesSelector = ({ activeKey, show, cookies, handleCategorySelection, handleSave, handleToggle }: Props) => (
```

Comment:
_(clears throat) I guess that's a form that we could have wrote with `react-final-form`_

Reply:
(PS: that's not a joke ;-) )